src/components/FooterWrapper.js

```diff
@@ -31,8 +31,11 @@ const FooterWrapper = ({ children }) => {
         footerLogo()
       ]),
       a({ href: Nav.getLink('privacy'), style: styles.item }, 'Privacy Policy'),
+      div({ style: styles.item }, '|'),
       a({ href: Nav.getLink('terms-of-service'), style: styles.item }, 'Terms of Service'),
       div({ style: styles.item }, '|'),
+      a({ href: 'https://support.terra.bio/hc/en-us/articles/360030793091-Terra-FireCloud-Security-Posture', target: '_blank', style: styles.item }, 'Security'),
```

Comment:
Nit: This should have a `'pop-out'` icon (like Documentation below) because it opens a new site in a new window.